### ISSUE TITLE
DBeaver should refresh Database Navigator after Cascade delete

### ISSUE THREAD

**Issue by mboeringa** [dbeaver/dbeaver#6396]
**Is your feature request related to a problem? Please describe.**
DBeaver currently does not seem to automatically refresh the Database Navigator's tree view if a table was deleted with the "Cascade" option. After deletion of the table, the dependent views are still shown, see attached screenshot, even though these objects no longer exist. Only after hitting F5, do these phantom objects disappear.

**Describe the solution you'd like**
DBeaver should refresh the Database Navigator in case of a "Cascade" delete.

**Describe alternatives you've considered**

**Additional context**
As similar issue, although not mentioning "Cascade" delete, is https://github.com/dbeaver/dbeaver/issues/1380

![afbeelding](https://user-images.githubusercontent.com/7635726/62274812-e5ee0a00-b440-11e9-9d6b-6d714f49ede0.png)

**Comment by uslss** [comment]
duplicate of #6398
